.github/workflows/links.yml

```
@@ -1,45 +1,37 @@
-  schedule:
-    - cron: '00 18 * * *'
+on: [push, pull_request]
 
 jobs:
-  linkChecker:
+  check-links:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)